resource/labeling-app/app.js

```diff
@@ -150,6 +181,15 @@ export default new Vue({
                                          :isFirst="isFirst || !(stats.labeled + stats.skipped)"
                                          :isLabeled="isCurrentItemLabeled()"/>
                     </div>
+
+                    <halting-criterion-metric
+                            v-if="isAlEnabled && type === 'image-object'"
+                            :thresholds="haltingThresholds"
+                            :colors="['#2AA876','#FFD265','#CE4D45']"
+                            :currentValue="item.data.enriched.halting"
+                            @currentColor="currentALColor=$event"></halting-criterion-metric>
```

Comment:
no update pattern ?

Reply:
actually currentALColor is not used in the end, I'll remove it